### ISSUE TITLE
Demo Deployment on Vercel not found

### ISSUE THREAD

**Issue by srvtotaldev** [nicholaschiang/poll#1]
Hi brother,

Vercel Demo of your script is down for a while -> https://poll.nicholaschiang.com/

![image](https://user-images.githubusercontent.com/121265418/209237427-ca69242a-bbdd-4fc4-823c-f51927e8f8ec.png)

**Comment by nicholaschiang** [comment]
yeah, i took it down a while ago cause it consistently went over Vercel's edge function invocation limits for free accounts.

if u wanna use it, i suggest creating a new vercel account and deploying it yourself. then it should work fine (it works by basically triggering a ton of edge function calls from the user's browser where each invocation tries to add a vote).

**Comment by srvtotaldev** [comment]
Thanks for letting me know.

I deployed it but couldn't get it to work -> https://poll-sand.vercel.app/

It runs but won't add votes to the poll


![image](https://user-images.githubusercontent.com/121265418/209291666-c2b94612-7430-4f50-84e3-899094d407d5.png)

![image](https://user-images.githubusercontent.com/121265418/209291750-ae42b0fa-50b7-44b2-968c-bb40c512f5f4.png)
